reducers/views/i18n.js

```diff
@@ -3,11 +3,9 @@
 
 import {combineReducers} from 'redux';
 
-import en from 'i18n/en.json';
-
 import {ActionTypes} from 'utils/constants.jsx';
 
-function translations(state = {en}, action) {
+function translations(state = {}, action) {
```

Comment:
I don't think the change in when the English translations are added should make a difference, but any idea of we might be using those translations before loadTranslations has been called?

Reply:
As far as I can tell we can't use the translations system before that point, so if there are instances, they don't work anyway.